src/App.css

```diff
@@ -2,12 +2,55 @@
   margin: 5px 0;
   width: 100%;
 }
-.btn-primary {
+.search-sidebar {
+  flex-grow: 3;
+  border: 1px solid lightgray;
+  max-width: 420px;
+}
+input.input-style {
+  border: 0;
+  border-bottom: 2px solid #333;
+  padding: 15px 0;
+  display: block;
+  font-size: 18px;
+  font-family: "Muli", sans-serif;
   width: 100%;
-  margin-left: 5px;
+  transition: 0.1s ease-in;
+  margin: 5px;
+}
+input:focus,
+input:valid {
+  border-bottom-color: darksalmon;
+  outline: none;
+}
+button.booking {
+  color: white;
+  background: #ff8416;
+  border: none;
+  border-radius: 5px;
+  width: 60px;
+  height: 30px;
+  box-shadow: 2px 2px 2px #9e9e9e;
+}
+ul.list-group.flightrow {
```

Comment:
this is not a good way to css 
ul.list-group.flightrow . 
you should use parent class to target child class